scan/src/reusable/InfoMobileCard.re

```diff
@@ -1,29 +1,46 @@
+type coin_amount_t = {
+  value: list(Coin.t),
+  hasDenom: bool,
+};
+
 type t =
-  | Address(Address.t, int)
+  | Address(Address.t, int, bool)
```

Comment:
Maybe it's worth having a variant type for this. Can even be 
```Address(Address.t, int, [ `account  | `validator ])```. Or at least we can add a comment explaining the meaning of the boolean. Up to you guys.